packages/gui/src/components/settings/SettingsIntegration.tsx

```diff
@@ -131,6 +135,9 @@ export default function SettingsIntegration() {
     }
   }, [topic, pairs, refreshBypassCommands]);
 
+  const readonlySkipValue =
+    bypassReadonlyCommands && topic && fingerprint ? bypassReadonlyCommands[topic][fingerprint] : false;
```

Comment:
This line crashes if `bypassReadonlyCommands[topic] === undefined`. Easily repro'd if you create a WC pairing without enabling the new bypass option and then select the newly paired dapp in the settings screen.

Reply:
```suggestion
  const readonlySkipValue =
    bypassReadonlyCommands && topic && fingerprint && bypassReadonlyCommands[topic]
      ? bypassReadonlyCommands[topic][fingerprint]
      : false;
```